target/linux/generic/pending-5.4/611-netfilter_match_bypass_default_table.patch

```diff
@@ -76,17 +76,17 @@ Signed-off-by: Felix Fietkau <nbd@nbd.name>
 -	acpar.thoff   = ip_hdrlen(skb);
 -	acpar.hotdrop = false;
 -	acpar.state   = state;
- 
+-
 -	WARN_ON(!(table->valid_hooks & (1 << hook)));
 -	local_bh_disable();
  	addend = xt_write_recseq_begin();
--	private = READ_ONCE(table->private); /* Address dependency. */
+-	private = rcu_access_pointer(table->private);
```

Comment:
What's removed here is just added a few lines above in the same patch again. I assume (without understanding the code) that the code should also be changed according to https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable-rc.git/commit/net/ipv4/netfilter/ip_tables.c?h=queue/5.4&id=b17244cebb24b30b886d46938cc5f798f7337574 there ...